config.yml

```
@@ -110,14 +110,15 @@ ROCK_MAPPINGS:
       EntityType: ContentChannelItem
     UniversalContentItem:
       EntityType: ContentChannelItem
+      ContentChannelId: [16, 28]
```

This should include news as well. 
```suggestion
      ContentChannelId: [16, 27, 28]
```